modules/tensor_mechanics/src/utils/RankFourTensor.C

```diff
@@ -24,7 +24,7 @@ void mooseSetToZero<RankFourTensor>(RankFourTensor & v)
 MooseEnum
 RankFourTensor::fillMethodEnum()
 {
-  return MooseEnum("antisymmetric symmetric9 symmetric21 general_isotropic symmetric_isotropic antisymmetric_isotropic general");
+  return MooseEnum("antisymmetric symmetric9 symmetric21 general_isotropic symmetric_isotropic antisymmetric_isotropic axisymmetricrz general");
```

Comment:
Okay, I will correct the indentation and the name and try committing again.


On Tue, Jun 2, 2015 at 11:57 AM, Daniel Schwen <notifications@github.com>
wrote:

> In modules/tensor_mechanics/src/utils/RankFourTensor.C
> <https://github.com/idaholab/moose/pull/5157#discussion_r31551215>:
>
> > @@ -24,7 +24,7 @@ void mooseSetToZero<RankFourTensor>(RankFourTensor & v)
> >  MooseEnum
> >  RankFourTensor::fillMethodEnum()
> >  {
> > -  return MooseEnum("antisymmetric symmetric9 symmetric21 general_isotropic symmetric_isotropic antisymmetric_isotropic general");
> > +  return MooseEnum("antisymmetric symmetric9 symmetric21 general_isotropic symmetric_isotropic antisymmetric_isotropic axisymmetricrz general");
>
> Yeah, definitely axisymmetric_rz!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/5157/files#r31551215>.
>